.env.local.example

```
@@ -1,13 +1,22 @@
 # To enable / disable self-hosting features.
-SELF_HOSTED=true
+SELF_HOSTED = true
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)